lib/src/default_index_store.rs

```diff
@@ -2007,12 +2034,12 @@ mod tests {
     #[test_case(true; "file")]
     fn index_empty(on_disk: bool) {
         let temp_dir = testutils::new_temp_dir();
-        let index = MutableIndexImpl::full(3, 16);
+        let mutable_segment = MutableIndexImpl::full(3, 16);
```

Comment:
Were all these variable renames supposed to be in the next commit?

Reply:
Move to the next commit.

I did that in this commit because `MutableIndexImpl` is no longer an `Index`, but it's better to split noisy bulk renaming parts to a separate commit.